election-api-scala/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.2
+sbt.version=1.7.2
```

Comment:
Interestingly I can't get it to work with anything _newer_ than 1.7.2 because of some dependency conflicts.